src/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
```

Comment:
I renamed this file to `index` because from the get-go I was confused as to which of your components was the final export. This is also a well known pattern, which doesn't actually impact the name of your component.

Reply:
gotcha, `index.js` means final export file and I don't have to have the same name as the component here! Thanks